### ISSUE TITLE
Can't initialize last version of the client

### ISSUE THREAD

**Issue by Tardo** [telegramdesktop/tdesktop#4531]
### Steps to reproduce
1. Upgrade to last version
2. Run
3. Crash

### Expected behaviour
Expected run telegram desktop on linux without pulse (have Alsa, as usual...)

### Actual behaviour
Program crash

### Configuration
**Operating system:**
Manjaro (i3 desktop)

**Version of Telegram Desktop:**
1.2.14

**Used theme**:
Vertex-Maia-Dark

<details><summary><b>Logs</b>:</summary>
$ ./Telegram 

(Telegram:15072): Gtk-WARNING **: Imposible encontrar el motor de temas en la ruta al _modulo: «adwaita»,

(Telegram:15072): Gtk-WARNING **: Imposible encontrar el motor de temas en la ruta al _modulo: «adwaita»,
AL lib: (WW) alc_initconfig: Failed to initialize backend "pulse"
AL lib: (EE) ALCcaptureAlsa_open: snd_pcm_hw_params_set_channels(self->pcmHandle, hp, ChannelsFromDevFmt(device->FmtChans, device->AmbiOrder)) failed: Argumento inválido
double free or corruption (fasttop)
Abortado (`core' generado)

</details>

**Comment by john-preston** [comment]
@Tardo Which version was working for you before?

Can you please test different alpha versions from http://github.com/telegramdesktop/tdesktop/releases page?

Like 1.2.6 (previous stable): https://github.com/telegramdesktop/tdesktop/releases/download/v1.2.6/tsetup.1.2.6.tar.xz

1.2.8 alpha: https://github.com/telegramdesktop/tdesktop/releases/download/v1.2.8/tsetup.1.2.8.alpha.tar.xz

1.2.10.alpha: https://github.com/telegramdesktop/tdesktop/releases/download/v1.2.10/tsetup.1.2.10.alpha.tar.xz

1.2.12.alpha: https://github.com/telegramdesktop/tdesktop/releases/download/v1.2.12/tsetup.1.2.12.alpha.tar.xz

**Comment by Tardo** [comment]
Hi @john-preston ... The results:
1.2.6 - Works Fine
1.2.8 - Works Fine
1.2.10 - Not Works
1.2.12 - Not Works

**Comment by john-preston** [comment]
@Tardo 1.2.9.alpha: https://github.com/telegramdesktop/tdesktop/releases/download/v1.2.9/tsetup.1.2.9.alpha.tar.xz

(just to be sure)

**Comment by Tardo** [comment]
@john-preston
1.2.9 - Not Works

**Comment by john-preston** [comment]
@Tardo Does it offer you to send crash report when you relaunch the app after the crash?

**Comment by Tardo** [comment]
@john-preston no... the only window that i can see is one that says that Telegram was not closed porperly...
![telegram_window](https://user-images.githubusercontent.com/731270/37782533-60d39f44-2df3-11e8-8e6e-c2bf5f61a2ff.png)

**Comment by john-preston** [comment]
@Tardo Even in the alpha version? That's sad :(

**Comment by Nephiel** [comment]
Same here. No Pulseaudio. Opening any version from 1.2.9 to 1.2.14 results in a crash:
>`AL lib: (WW) alc_initconfig: Failed to initialize backend "pulse"`
>`AL lib: (EE) ALCcaptureAlsa_open: snd_pcm_hw_params_set_channels(self->pcmHandle, hp, ChannelsFromDevFmt(device->FmtChans, device->AmbiOrder)) failed: Invalid argument`
>`*** Error in 'Telegram': double free or corruption (fasttop): 0x0000000006a755a0 ***`

**Comment by Nephiel** [comment]
Attaching log for 1.2.14: [telegram.log](https://github.com/telegramdesktop/tdesktop/files/1842578/telegram.log)

**Comment by louwers** [comment]
No version of Telegram opens for me anymore after the update. It just hangs. No output.

**Comment by lightwavefeesh** [comment]
ALSA-only here, too. Same error log as @Nephiel. It'll start fine if I use a basic asoundrc (multiple I/O iirc) but the one I currently use (for loopback recording) stops [Telegram] dead.

**Comment by Francesco149** [comment]
same issue here on 1.2.15, here's the report https://gist.github.com/Francesco149/314be840704f1e6cff86e3c1f16021a9

**Comment by Tardo** [comment]
no fixes? :(

**Comment by Francesco149** [comment]
for now the only solution is using 1.2.6, stopping the update on startup and disabling updates from the settings.

or compile from source and disable updates like on gentoo's ebuild

**Comment by john-preston** [comment]
@Francesco149 I'll try to reproduce that on Manjaro. I don't know what caused the regression, so I'll have to just test a bunch of builds narrowing down the problem.

Also I could build 1.2.6 from source on my current build environment and ask someone to check if it still works or I've broken something in the build environment. For anyone who is ready to check please contact me at https://t.me/preston

**Comment by john-preston** [comment]
@Tardo No :( I'll try to reproduce that, please share the exact OS version (better with a link to an ISO image) that I could download and install in Parallels so that it would reproduce. I've tried Manjaro from here: https://manjaro.org/get-manjaro/ and it works fine there.

**Comment by Tardo** [comment]
@john-preston  This is the distribution that I use https://osdn.net/dl/manjaro-community/manjaro-i3-17.1.8-stable-x86_64.iso
